charts/rancher-cis-benchmark-crd/6.1.0-rc1/templates/clusterscanreport.yaml

```
@@ -0,0 +1,39 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterscanreports.cis.cattle.io
+spec:
+  group: cis.cattle.io
+  names:
+    kind: ClusterScanReport
+    plural: clusterscanreports
+  scope: Cluster
```

And this with https://github.com/rancher/cis-operator/blob/master/crds/clusterscanreport.yaml.
```suggestion
    plural: clusterscanreports
    singular: clusterscanreport
  scope: Cluster
```

---

done